GOVERNANCE.md

```diff
@@ -21,16 +21,17 @@ future addition of committee seats), the Envoy core proxy maintainer seat will b
 tie with a super vote.
 
 ## Steering Committee
-The steering committee will be a 5-member body that will make directional and any strategic
+The steering committee will be a 6-member body that will make directional and any strategic
 technical decisions on the Envoy gateway project. It will also decide on any changes on its
 governance, marketing and future direction. Seats will be tied to organizations and not
 individuals and will be valid for 1 year from the start date.
 
-- Envoy core proxy maintainers [Member: Matt Klein; Start Date: 5/16/2022]
-- Tetrate [Member: Varun Talwar; Start Date: 5/16/2022]
-- VMware [Member: Winnie Kwon; Start Date: 11/10/2022]
-- Ambassador Labs [Member: Richard Li; Start Date: 5/16/2022]
-- Fidelity Investments [Member: Venkat Kasisomayajula / Rajarajan Pudupatti Sundari Jeyakodi; Start Date: 5/16/2022]
+- Envoy core proxy maintainers [Member: Matt Klein; End Date: 5/16/2023]
```

Comment:
nit: 

> [Organizations can also be removed from the steering committee due to lack of project commitment
> on the same yearly basis. This will be determined by a majority vote of the existing committee.
> __The exception to this is the Envoy core proxy maintainer seat which is a permanent seat.__](https://github.com/envoyproxy/gateway/blob/main/GOVERNANCE.md?plain=1#L41) 

:slightly_smiling_face:

Reply:
Not sure how to resolve this one, just remove the end date of Envoy core proxy maintainers?

Reply:
Yes, that's fine. Thanks!

Reply:
Done @richarddli